### DISCUSSION TITLE
Suricata-start ---- state not allowed

### DISCUSSION THREAD

**Discussion by wege2003** [Security-Onion-Solutions/securityonion#3611] [Unsupported Versions] [answered]
Hi there,
i have following problems .... suricata wont start (Screenshot)
How can i fix this error?

Greets
LaviS
![so-suricata](https://user-images.githubusercontent.com/31622452/112300542-4165af00-8c99-11eb-9ef0-a430dcc18a23.png)

**Comment by weslambert** [comment, marked as answer]
What kind of node are you trying to run this from?  Normally this happens when you are trying to start Suricata on a node that is not supposed to be running Suricata.
